src/briefcase-forms.rst

```diff
@@ -57,98 +101,128 @@ Pulling forms from Collect
-- Open *ODK Briefcase*.
-- In the window which opens up select :guilabel:`pull` tab.
-- Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
-- Select the unzipped :file:`odk` folder that you copied onto your local hard drive.
-- Click on :guilabel:`pull` button at the bottom of the window and the filled-in submissions that were copied off of the device will be loaded into ODK Briefcase's storage location.
-- On the Android device, open ODK Collect and delete the filled-in forms.
+Current Android devices (those running v4.x or later) use a USB mounting protocol that Briefcase does not support. Therefore, the procedure for getting form instances out of Collect is different.
 
+#. Ensure all filled-in forms are finalized. 
 
-.. tip::
- - You can use the *Custom path to ODK Directory* any time you want to pull forms from custom location.
- - You can confirm that the forms have been successfully pulled into ODK Briefcase by confirming a successful pull status or by verifying the data appearing in a CSV export file.
+   If you have incomplete forms that you cannot finalize before pulling into Briefcase, delete them. If you need to keep them, make a copy :file:`/sdcard/odk` before deleting them, and restore it after you are finished.
+
+#. Create a zip archive of the entire :file:`odk` directory.
+
+   .. tip::
+   
+     You'll need to use an app for this. 
+     
+     One option is `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
+     
+#. Connect your Android device to your computer using a USB cable and choose to mount it as a Media device.
+#. Copy the zip file you created from the Android device to your local hard drive.
+#. Once it is copied onto your local hard drive, unzip the file.
+#. In Briefcase, open the :guilabel:`Pull` tab.
+#. Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
+#. Select the unzipped :file:`odk` folder.
+#. Click :guilabel:`Pull`.
+#. On the Android device, open ODK Collect and delete the filled-in forms.
+
+
+   .. tip::
+  
+     - You can use the *Custom path to ODK Directory* any time you want to pull forms from custom location.
+     - You can confirm that the forms have been successfully pulled into Briefcase by confirming a successful pull status or by verifying the data appearing in a :ref:`CSV export file <briefcase-export-to-csv>`.
 
-.. warning::
- - ODK Briefcase does not work with the USB-mounted Media device (MTP) protocol, which replaced the USB-mounted drive protocol on Android 4.x devices.
- - ODK Briefcase does not support pushing blank forms to ODK Collect. You can manually install the forms on your ODK Collect device.
+     
+.. note:: 
 
+  ODK Briefcase does not support pushing blank forms to ODK Collect. Instead, :ref:`manually load the forms on your ODK Collect device <loading-forms-directly>`.
 
-.. _export-forms:
+
+.. _briefcase-export-to-csv:
 
 Export forms to CSV
```

Comment:
Briefcase 1.9.0 has customizable exports of form if you hit the gear button after you pull. There should be language about that.